lib/Form/EditContactGroup.php

```diff
@@ -36,9 +36,16 @@ public function __construct($vars, $contact)
         }
     }
 
-    public function renderActive($renderer, $vars, $action, $method)
+    public function renderActive(
+        $renderer = null,
+        $vars = null,
+        $action = '',
+        $method = 'get',
+        $enctype = null,
+        $focus = true
+    )
     {
-        parent::renderActive($renderer, $vars, $action, $method);
+        parent::renderActive($renderer, $vars, $action, $method, $enctype, $focus);
```

Comment:
Forward the additional parameters to parent class. As they likely always have the default values, it does not break existing calls.